packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -176,4 +179,27 @@ export class FlowBuilderApi {
   private containsAnyKeywords(input: string, keywords: string[]): boolean {
     return keywords.some(keyword => input.includes(keyword))
   }
+
+  getPayloadFromBotActionNodeId(id?: string): string | undefined {
+    if (id) {
+      const botActionNode = this.getNodeById(id)
```

Comment:
I think the name of this variable does not represents what it is as this node could be of another type, not a bot action one...

Reply:
What I was trying to explain here is that you were getting the node and then after that there was an `if` checking if the node is of type BotAction, so it meant that it couod be the possibility that it was not of type BotAction.